### PR TITLE
Various changes to move towards lessening configuration

### PR DESCRIPTION
**Pull request by pgm**
In a recent conversation @5im1z and I discussed how to handle per-environment configuration. My stance is that less configuration is easy to work with, and it's good to make any configuration concise and explicit so that it's easy to see what's actually different between environments. We also talked about, in general, the issue of how to manage different configurations.

I have a specific set of ideas in mind around refactors for configuration, but in conversations I feel like it's always been fairly abstract, so I created this branch to demonstrate what the sort of a refactor I have in mind would look like.

I created this branch with the following goals in mind:

1. These changes should be _only_ mechanical refactorings. No functionality has changed. (For example, the behavior where a different configuration is chosen based on the git branch is still in place even though I'd like to removed that ultimately.)
2. Be explicit about variables which do not change, and put them all in a module called `constants` to make it clear these do not change.
3. If a variable is the same in _both_ prod and dev configurations, it should also be in `constants`
4. Remove all `from depmapomics.config import *` to make it easier to see which variables are from which modules. These have all been replaced with `from depmapomics import constants` and `from depmapomics import env_config`. 
5. Leave the code in a place which will make it *easier* to change configuration from a global module to an object which is passed to the methods that need it in a _future_ refactor.

After making these changes, I can now grep for `env_config` and see there's actually only *3* files which *actually* depend on the env_config. Now, making a round to update just these functions to remove the dependence on `env_config` feels like a much more tractable change then having to update all files.

### CODE REVIEW

File: depmapomics/qc/config.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from depmapomics import constants
```
Comment (5im1z) [inline depmapomics/qc/config.py:1]: constants is imported but unused

File: depmapomics/qc/conftest.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
+from depmapomics import constants
```
Comment (5im1z) [inline depmapomics/qc/conftest.py:1]: imported but unused, same for all other modules in depmapomics/qc

File: depmapomics/qc/rna.py [hunk 3]
```diff
@@ -1,10 +1,11 @@
+from depmapomics import constants
 import sys
 
 import pandas as pd
 from genepy import rna
 from depmapomics import terra
 import numpy as np
-from depmapomics.config import RNASEQC_THRESHOLDS_FAILED, RNASEQC_THRESHOLDS_LOWQUAL
+from depmapomics import constants
```
Comment (5im1z) [inline depmapomics/qc/rna.py:8]: here it is actually used, but import is duplicated

**Review by 5im1z — COMMENTED** [review]
Thanks for taking this on! Looks good to me, besides some unused imports in the QC module. There are quite a few parameters that are outdated or can be isolated and moved to the upload repo. I will work on cleaning up after merging!

File: depmapomics/qc/config.py (hunk 1 above)
Comment (pgm) [reply depmapomics/qc/config.py]: Removed

File: depmapomics/qc/conftest.py (hunk 2 above)
Comment (pgm) [reply depmapomics/qc/conftest.py]: removed

File: depmapomics/qc/rna.py (hunk 3 above)
Comment (pgm) [reply depmapomics/qc/rna.py]: removed

**Review by 5im1z — APPROVED** [review]
looks good!

**Pull request merged**